src/app/pages/data-protection/scrub/resilver-config/resilver-config.component.html

```diff
@@ -0,0 +1,42 @@
+<mat-card class="card">
+  <mat-card-content>
+    <mat-progress-bar *ngIf="isFormLoading" mode="indeterminate"></mat-progress-bar>
+    <form [formGroup]="form" class="ix-form-container" (submit)="onSubmit()">
+      <ix-fieldset [title]="'Resilver Priority' | translate">
+        <ix-checkbox
+          formControlName="enabled"
+          [label]="'Enabled' | translate"
+          [tooltip]="tooltips.enabled"
+        ></ix-checkbox>
+        <ix-select
+          formControlName="begin"
+          [label]="'Begin' | translate"
+          [options]="timeOptions$"
+          [tooltip]="tooltips.begin"
+        ></ix-select>
+        <ix-select
+          formControlName="end"
+          [label]="'End' | translate"
+          [options]="timeOptions$"
+          [tooltip]="tooltips.end"
+        ></ix-select>
+        <ix-select
+          formControlName="weekday"
+          [label]="'Days of the Week' | translate"
+          [options]="daysOfWeek$"
+          [tooltip]="tooltips.weekday"
+          [multiple]="true"
+        ></ix-select>
```

Comment:
Could you add `[required]="true"` ? Please confirm the required select in the previous version.

Reply:
It was, good catch.